components/icons/images/tier2/edit-not-editable.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <path fill="#494c4e" stroke="#494c4e" stroke-linecap="round" stroke-width="2" d="M2.5 4.5l17 17" opacity=".998"/>
+  <path fill="#494c4e" d="M15.62 12.619L14.206 11.2l4.586-4.586-1.412-1.407-4.58 4.587-1.42-1.415 6-6 4.24 4.24-6 6zm7.085-7.084l-4.241-4.243.706-.707a2 2 0 012.83 0L23.413 2a2 2 0 010 2.828l-.706.708z"/>
+  <path fill="#494c4e" d="M.56 24a.584.584 0 01-.19-.031.667.667 0 01-.18-.16.679.679 0 01-.16-.18.587.587 0 010-.37l2.15-4.94.671-1.409 6-6 1.416 1.416-5.757 5.755L4 19.149l-.654 1.506L4.852 20l1.068-.508 5.755-5.756 1.414 1.414-6 6-1.41.67L.74 23.969A.569.569 0 01.56 24z"/>
+</svg>
```

Comment:
Whoa, so much cleaner/smaller now! 🎉

Reply:
One other question for the designer maybe... I see an opacity of `.998` -- was that intentional? Seems so slight that it may have been a mistake.

Reply:
Yes that is a mistake, fixed it now